Marlin/src/pins/pins_BIGTREE_SKR_PRO_V1.1.h

```diff
@@ -219,7 +219,7 @@
   #endif
 
   // Alter timing for graphical display
-  #if HAS_GRAPHICAL_DISPLAY
+  //#if HAS_GRAPHICAL_DISPLAY
```

Comment:
I can't found where `HAS_GRAPHICAL_DISPLAY` defined， now it will nerve be compiled

Reply:
It should be `HAS_GRAPHICAL_LCD` instead of `HAS_GRAPHICAL_DISPLAY`.

Reply:
> It should be `HAS_GRAPHICAL_LCD` instead of `HAS_GRAPHICAL_DISPLAY`.

maybe, let me update it